deploy/srep-1000-ethtool-exporter/config.yaml

```
@@ -0,0 +1,14 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+    - key: ext-hypershift.openshift.io/cluster-type
+      operator: In
+      values: ["management-cluster"]
+    - key: api.openshift.com/fedramp
+      operator: NotIn
+      values: ["true"]
+    - key: api.openshift.com/environment
```

It's actually "stage" and "integration". Just use NotIn to keep it simple.
```suggestion
    - key: api.openshift.com/environment
    operator: NotIn
    values:
      - "production"
```